amm/src/test/resources/bsp/multi/lib3.sc

```diff
@@ -0,0 +1,5 @@
+// See https://github.com/Baccata/gradle-simple/blob/master/src/main/java/Hello.java
+import $repo.`https://jitpack.io`
+import $ivy.`com.github.baccata:gradle-simple:63f75f7133`
```

Comment:
needed a java library to avoid having to cross-compile to various Scala versions.

The original https://github.com/jitpack/gradle-simple only has not-package class containing a main method. I'm not sure why, but ammonite didn't seem to be happy with it. Forked it to add package + remove main. 

I'm happy to transfer the repo's ownership to whatever relevant github organisation (coursier or haoyi's) if you guys prefer

Reply:
Replaced in 68d71e0